src/UvDriver.php

```diff
@@ -57,7 +57,7 @@ public function open(string $path, string $mode): Promise
 
         $openArr = [$mode, $path, $deferred];
         \uv_fs_open($this->loop, $path, $flags, $chmod, function ($fh) use ($openArr): void {
-            if ($fh) {
+            if (is_resource($fh)) {
```

Comment:
With ext-uv 0.2 `$fh` is `resource|false`.
With ext-uv 0.2 `$fh` is `resource|int` (It was -17 to be exact).

I've no idea what the integer means but most likely some kind of a failure.

I'm wondering if we should use `is_resource` in other cases as well but I'm really way over my head with these internal things...

Reply:
Yes, I guess we need that everywhere then.